features/runner/developer_skips_specs.feature

```diff
@@ -36,6 +36,10 @@ Feature: Developer skips examples
       }
 
       """
-    When I run phpspec using the "dot" format
+    When I run phpspec with the "verbose" option
```

Comment:
Can you please split this into a new scenario specifically about the verbose mode behaviour (that doesn't need to have 'the suite will pass' as a step)

Reply:
makes sense, I added a whole new scenario :+1: